client/app/bundles/course/assessment/submission/components/SubmissionAnswer.jsx

```diff
@@ -128,7 +128,7 @@ class SubmissionAnswer extends Component {
     const answersLoaded = historyQuestion ? historyQuestion.pastAnswersLoaded : false;
     const isLoading = historyQuestion ? historyQuestion.isLoading : false;
     const isAutograding = questionsFlags[id] ? questionsFlags[id].isAutograding : false;
-    const disabled = noPastAnswers || isLoading || isAutograding;
+    const disabled = historyQuestion === undefined || noPastAnswers || isLoading || isAutograding;
```

Comment:
`noPastAnswers` should have covered this condition. The actual bug is a few lines above

```
 const noPastAnswers = historyQuestion ? historyQuestion.answerIds.length === 0 : false
```

If there is no `historyQuestion`, then `noPastAnswers` should be `true`